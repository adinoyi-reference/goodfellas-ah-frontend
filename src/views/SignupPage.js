import React from 'react';
import SignupForm from '../containers/SignupForm';
import Auth from '../components/layout/Auth';


const Signup = (props) => (
  <div>
    <Auth><SignupForm {...props} /></Auth>
  </div>
);

export default Signup;
