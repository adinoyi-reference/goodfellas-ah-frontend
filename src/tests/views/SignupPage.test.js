import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SignupPage from '../../views/SignupPage';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<SignupPage />);
});

describe('SignupPage UI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
