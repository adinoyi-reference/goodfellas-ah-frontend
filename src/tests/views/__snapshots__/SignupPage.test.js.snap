// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignupPage UI render features view should render as expected 1`] = `
<div>
  <Auth>
    <Connect(SignupForm) />
  </Auth>
</div>
`;
